tbx/static_src/sass/components/_footer.scss

```diff
@@ -91,4 +91,55 @@
             filter: none;
         }
     }
+
+    &__carbon-impact-container {
+        background-color: var(--color--footer-background);
+        border-top: 1px solid $color--grey-80;
```

Comment:
We aren't using sass variables anywhere for colours in this build. This may well need to be a different colour for light mode. If this definitely needs to be a new colour, rather than re-using an existing border colour, could you add it as a css variable in the `.mode-dark` section in `variables.scss`.

Reply:
For the sake of pragmatism I've removed this new variable and re-used the existing border colour for consistency, as the difference is minimal